components/playlist/browser/resources/components/playlistItem.tsx

```diff
@@ -6,34 +6,91 @@
 import * as React from 'react'
 import styled, { css } from 'styled-components'
 
-import ThumbnailIcon from '../assets/playlist-thumbnail-icon.svg'
+import { color, font } from '@brave/leo/tokens/css'
+import Icon from '@brave/leo/react/icon'
+import DefaultThumbnailIcon from '../assets/playlist-thumbnail-icon.svg'
 
 interface Props {
-    id: string
-    name: string
-    thumbnailUrl: string
+  id: string
+  name: string
+  thumbnailUrl: string
+  cached: boolean
 
-    onClick: (id: string) => void
+  onClick: (id: string) => void
 }
 
-const StyledThumbnail = styled.img<{showDefaultThumbnail: boolean}>`
-  max-width: 100%;
+const ThumbnailStyle = css`
+  flex: 0 0 140px;
+  height: 80px;
+  object-fit: cover;
+  border-radius: 4px;
+`
+const StyledThumbnail = styled.img`
+  ${ThumbnailStyle}
+`
+
+const DefaultThumbnail = styled.div`
+  ${ThumbnailStyle}
+  content: url(${DefaultThumbnailIcon});
+`
+
+const PlaylistContainer = styled.li`
+  display: flex;
+  padding: 8px 16px 8px 8px;
+  align-items: center;
+  gap: 16px;
+  align-self: stretch;
+`
+
+const ItemInfoContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 4px;
+`
+
+const ItemName = styled.span`
+  color: ${color.text.primary};
+  font: ${font.primary.xSmall.regular};
+  cursor: default;
```

Comment:
Is this needed still? (is it undoing a cursor from something further up the tree?)